src/Chapter3.hs

```diff
@@ -971,6 +1134,31 @@ implement the following functions:
 🕯 HINT: to implement this task, derive some standard typeclasses
 -}
 
+data Day
+    = Monday
+    | Tuesday
+    | Wednesday
+    | Thursday
+    | Friday
+    | Saturday
+    | Sunday
+    deriving (Show, Eq, Enum, Bounded)
+
+isWeekend :: Day -> Bool 
+isWeekend day
+    | elem day [Friday, Saturday, Sunday] = True 
```

Comment:
I think `Friday` is not a part of the weekend 🤔 
But otherwise, the function definition looks good 👍🏻 

The only improvement I can think of is that `elem` already returns `Bool`, so you don't need guards, and can write just:

```haskell
isWeekend day = elem day [Saturday, Sunday]
```